main.tex

```diff
@@ -268,20 +268,22 @@ \section*{Abstract} %(301 out of 300 words)
 %3) What is the methodology being used to answer the stated research question? (81 words)
 Therefore, we exploit the provenance data model, PROV-DM, of the World Wide Web Consortium and, building on previous work, continue developing a PROV ontology for simulation studies.
 Based on a case study of 19 \wnt{}/\bcat{} signaling models, we identify crucial entities and activities as well as useful metadata to both capture the provenance information of individual simulation studies and relate these forming a family of models.
-The approach is implemented in \href{https://github.com/SFB-ELAINE/WebProv}{\webprov{}}, which allows one to insert and query provenance information.
+The approach is implemented in \href{https://github.com/SFB-ELAINE/WebProv}{\webprov{}}, a web application that allows one to insert and query provenance information.
 
 %4) What are the results obtained? (73 words)
-Our specialization of PROV-DM contains the entities \RQ{}, \A{}, \R{}, \QM{}, \SM{}, \SE{}, \SD{}, and \WD{} as well as activities referring to building, calibrating, validating, and analyzing a simulation model.
+Our specialization of PROV-DM contains the entities \RQ{}, \A{}, \R{}, \QM{}, \SM{}, \SE{}, \SD{}, and \WD{} as well as activities that include building, calibrating, validating, and analyzing a simulation model.
 We show that most \wnt{} simulation models are connected to other \wnt{} models by using (parts of) these models.
 However, the overlap, especially regarding the \WD{} used for calibration or validation of the models is small.
 
 %5) What is the meaning and importance of these results? (52 words)
 Making these aspects of developing a model explicit and queryable is an important step for assessing and reusing simulation models more effectively.
-By this, it helps to integrate a new simulation model within a family of existing ones.
+Exposing this information helps to integrate a new simulation model within a family of existing ones.
 Our approach may lead to the development of more robust and valid simulation models.
+% TODO combine above?
 
 %6) What are the directions for follow-up research?  (24 words)
 We hope that it becomes part of a standardization effort and that modelers adopt the benefits of provenance when considering or creating simulation models.
```

Comment:
What do you mean by "it" here?

Reply:
it = our approach of using PROV-DM for simulation models. Should I use "our approach" or "this approach" or sth. else?

Reply:
Yes I think "our approach" would clearer :)